src/__experimental__/components/search/search.style.js

```diff
@@ -1,17 +1,31 @@
 import styled, { css } from "styled-components";
 import StyledInputPresentation from "../input/input-presentation.style";
+import StyledInput from "../input/input.style";
 import StyledIcon from "../../../components/icon/icon.style";
 import StyledButton from "../../../components/button/button.style";
 import { baseTheme } from "../../../style/themes";
 import StyledFormField from "../form-field/form-field.style";
 
 const StyledSearch = styled.div`
-  ${({ isFocused, searchWidth, searchIsActive, searchHasValue, theme }) => {
+  ${({
+    isFocused,
+    searchWidth,
+    searchIsActive,
+    searchHasValue,
+    searchButton,
+    theme,
+    variant,
+  }) => {
+    const darkTheme = variant === "dark";
```

Comment:
This should probably be `isDarkVariant` too.
```suggestion
    const isDarkVariant = variant === "dark";
```

Reply:
Yep. Good point. I'll get this changed too.